recipe/meta.yaml

```diff
@@ -20,12 +20,19 @@ requirements:
     - pip
   run:
     - python >=3.6
+    - cartopy
     - cftime ==1.0.1
+    - cf_units
     - clize
+    - dask
     - iris >=2.2
+    - netCDF4
     - numpy
+    - python-dateutil
     - python-stratify
+    - pytz ==2020.5
```

Comment:
Similar question about having pytz be optional as it is only used for ancillary files

Reply:
See here: https://github.com/metoppv/improver/pull/1436#discussion_r603009495

Reply:
Thanks, makes sense. I did not realise timezonefinder (also only used for generating ancillary files) is only imported in a function.